Assets/Scripts/Models/Furniture.cs

```diff
@@ -55,8 +56,6 @@ public class Furniture : IXmlSerializable, ISelectable, IContextActionProvider,
     /// </summary>
     private Parameter furnParameters;
 
-    private float powerValue;
```

Comment:
This has broken the power overlay. #1147 I don't know the new power system well enough... what property is to be used in its place?